lua/nvim-treesitter/textobjects/lsp_interop.lua

```diff
@@ -46,6 +46,9 @@ function M.preview_location(location, context)
 
   local config = configs.get_module "textobjects.lsp_interop"
   local opts = {}
+  if config.max_height > 0 then
```

Comment:
couldn't you just forward all module setting to `open_floating_preview` or create a setting or create a `floating_preview_opts` key? You could key the border option for backward compatibility

Reply:
I like the `floating_preview_opts` key idea, more general. Will change it soon.